infra/bisector.py

```diff
@@ -113,39 +112,52 @@ def bisect(commit_old, commit_new, testcase, fuzz_target, build_data):
 
   Returns:
     The commit SHA that introduced the error or None
+
+  Raises:
+    Value Error: when a repo url can't be determine from the project
   """
   local_store_path = tempfile.mkdtemp()
-  repo_url = build_specified_commit.infer_main_repo(build_data.project_name,
-                                                    local_store_path,
-                                                    commit_old)
-  bisect_repo_manager = repo_manager.RepoManager(repo_url, local_store_path)
+  repo_url, repo_name = build_specified_commit.infer_main_repo(
+      build_data.project_name, commit_old)
+  if not repo_url or not repo_name:
+    raise ValueError('Main git repo url can not be determined.')
```

Comment:
`local_store_path` is not deleted on exception. Let's use the tempfile.TemporaryDirectory() context manager here instead.